pkg/flowser/app.go

```diff
@@ -0,0 +1,207 @@
+package flowser
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/google/go-github/github"
+)
+
+const (
+	darwin  = "darwin"
+	windows = "windows"
+)
+
+// New create new Flowser application.
+func New() *App {
+	return &App{}
+}
+
+type App struct {
+	installDir string
+}
+
+var errorPlatformNotSupported = errors.New("OS not supported, only supporting Windows and Mac OS")
+
+// Run starts the Flowser application with provided path to the flow project.
+//
+// Project path if exists should be set to the folder containing flow.json, if no such path exists pass empty value.
+func (a *App) Run(installDir string, projectPath string) error {
+	exe, err := a.executable(installDir)
+	if err != nil {
+		return err
+	}
+
+	if projectPath != "" {
+		projectPath = fmt.Sprintf("--project-path=%s", projectPath)
+	}
+
+	return exec.
```

Comment:
I noticed that the child process (Flowser) keeps running on Windows after you kill the parent (go program).

I'm not sure why that's the case, but it seems to be the default behavior on Windows. Here is the orphan process info:
![image](https://user-images.githubusercontent.com/36109955/202806965-9bdb61ef-0ba7-47a8-aad9-2a0a16f774cf.png)

Reply:
This is probably not the highest priority, so we can fix this at some later point.

Reply:
Very good point!